test/cpp/cpp_test_util.h

```diff
@@ -32,7 +32,11 @@ bool CloseValues(at::Tensor tensor1, at::Tensor tensor2, double rtol = 1e-5,
 
 static inline void AllClose(at::Tensor tensor, at::Tensor xla_tensor,
                             double rtol = 1e-5, double atol = 1e-8) {
-  EXPECT_TRUE(CloseValues(tensor, xla_tensor, rtol, atol));
+  if (at::isIntegralType(tensor.scalar_type())) {
```

Comment:
This is a tempting shortcut, but does not make a lot of sense to have an API take tolerances when we know upfront (from the test) that we are testing integers.

Reply:
I agree in principle, I just didn't like the extent of changes if I inlined `EXPECT_TRUE(EqualValues(...))` instead. Would you rather have that instead, maybe with a helper to hide `EXPECT_TRUE`?

Reply:
Hmm, `AllClose()` used to do more stuff (ToTensor, ...) but now is a simple wrapper over `EXPECT_TRUE()`.
On one side it is tempting to just use `EXPECT_TRUE()` inlined, on the other side we might lose some flexibility if we later want to add more stuff in there.
I'd just have an `AllCloseIntegrals()` at this point.

Reply:
Rather `AllEqualIntegrals()`, integrals can't really be "close".

Reply:
Yeah, sure. I mistyped it.

Reply:
Done.